docs/source/evaluation_suite.mdx

```diff
@@ -0,0 +1,48 @@
+# Creating an EvaluationSuite
+
```

Comment:
What do you think about taking a step back at the very beginning and motivating why evaluating models on multiple datasets with multiple metrics is a good idea?

Reply:
Added!